subprojects/dependency-management/src/main/java/org/gradle/internal/management/DefaultVersionCatalogBuilderContainer.java

```diff
@@ -76,6 +77,13 @@ private static void validateName(String name) {
         if (!VALID_EXTENSION_PATTERN.matcher(name).matches()) {
             throw new InvalidUserDataException("Invalid model name '" + name + "': it must match the following regular expression: " + VALID_EXTENSION_NAME);
         }
+
+        if (!name.equals("libs") && !name.endsWith("Libs")) {
+            DeprecationLogger.deprecateBehaviour("The name of version catalogs must end with 'Libs' to reduce chances of extension conflicts.")
+                .willBeRemovedInGradle8()
```

Comment:
Isn't this `willBecomeAnErrorInGradle8()`?

Reply:
🤦